client/app.jsx

```diff
@@ -33,6 +34,7 @@ const theme = createTheme({
 function App() {
   const [userId] = useState(0);
   const [geoLocation, setGeoLocation] = useState('');
+  const [route, setRoute] = useState('');
```

Comment:
Instead of naming this `route`/`setRoute` (which is usually something with a URL), I recommend renaming it to `pageTitle` or `headerText`.